src/patternfly/components/Form/examples/Form.md

```diff
@@ -130,6 +140,7 @@ cssPrefix: pf-c-form
 | `.pf-c-form__actions` | `<div>` | Iniates a row of actions. |
 | `.pf-m-action` | `.pf-c-form__group` | Modifies form group margin-top. |
 | `.pf-m-error` | `.pf-c-form__helper-text`| Modifies text color of helper text. |
+| `.pf-m-validated` | `.pf-c-form__helper-text` | Modifies text color of helper text. |
```

Comment:
what is the reason for the name being `validated` over something like `success` just because that would match the color name that it is using. I also think that the term "validated" is dependent on each product that the form is in.. i.e. one validation may have passed, so its successful, but it might not have passed a second validation on submit or something... maybe im overthinking it, but with naming i think keeping it as open as possible is better.

Reply:
This works for me. I think "success" covers something being validated, and can be used outside of the context of validated to indicate a success state. And it matches the alert and progress components. @mceledonia @mcarrano what do you think about using a more generic "success" modifier name instead of "validated"?

Reply:
I could also see using "valid" since the error state is styled via `aria-invalid`

Reply:
My original thought was this was really only to be used in the case where the input value has been `validated`, not just `valid`. Most input values that are not `invalid` would thus be `valid`. However, `validated` has a different meaning and should only be used in cases where there was some effort involved in validation, not just `yeah, that is a number`.

Reply:
I like calling this 'success' to be consistent with alert states. Agree with @jeff-phillips-18 that 'valid' does not feel right.

Reply:
@christiemolloy updated to be `pf-m-success`